README.md

```diff
@@ -126,45 +99,90 @@ With
 
 and `path_output_folder`: path of the folder where simulation results should be stored.
 
-For more information about the possible command lines
+For more information about the possible command lines options
 
-`python mvs_tool.py -h`
+    `mvs_tool -h`
 
-##### 2) Use the `main()` function
+##### Use the `main()` function
 
-Edit the csv files (or, for devs, the json file) and run the `main()` function. The following `kwargs` are possible:
+You can also execute the mvs within a script, for this you need to import
 
-- `overwrite` (bool): Determines whether to replace existing results in `path_output_folder` with the results of the current simulation (True) or not (False). Default: `False`.
-- `input_type` (str): Defines whether the input is taken from the `mvs_config.json` file ("json") or from csv files ('csv') located within <path_input_folder>/csv_elements/. Default: `json`.
-- `path_input_folder` (str): The path to the directory where the input CSVs/JSON files are located. Default: `inputs/`.
-- `path_output_folder` (str): The path to the directory where the results of the simulation such as the plots, time series, results JSON files are saved by MVS E-Lands. Default: `MVS_outputs/`.
+```
+from multi_vector_simulator.cli import main
+
+```
+The possible arguments to this functions are:
+- `overwrite` (bool): Determines whether to replace existing results in `path_output_folder` with the results of the current simulation (True) or not (False) (Command line "-f"). Default: `False`.
+- `input_type` (str): Defines whether the input is taken from the `mvs_config.json` file ("json") or from csv files ('csv') located within <path_input_folder>/csv_elements/ (Command line "-ext"). Default: `json`.
+- `path_input_folder` (str): The path to the directory where the input CSVs/JSON files are located. Default: `inputs/` (Command line "-i").
+- `path_output_folder` (str): The path to the directory where the results of the simulation such as the plots, time series, results JSON files are saved by MVS E-Lands (Command line "-o"). Default: `MVS_outputs/`.
 - `display_output` (str): Sets the level of displayed logging messages. Options: "debug", "info", "warning", "error". Default: "info".
 - `lp_file_output` (bool): Specifies whether linear equation system generated is saved as lp file. Default: False.
+- `pdf_report` (bool): Specify whether pdf report of the simulation's results is generated or not (Command line "-pdf"). Default: False.
+- `param save_png` (bool): Specify whether png figures with the simulation's results are generated or not (Command line "-png"). Default: False.
```

Comment:
`param save_png` or `param_save_png`?

Reply:
I believe `save_png`, param is probably a reminder from the docstring automatic formatting that was default on my pycharm

Reply:
Done in c06d115b309523351c3210b5e566667625ba93b0